contents/docs/deployment/deploy-source.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Deploying to Source
+sidebar: Docs
+showTitle: true
+---
+
+## Why Source Install
+
+To install from source, you need to build the source code and take care of the dependencies yourself. Likewise, you also need to be aware of all the features of the package so that you can build it accordingly.
+
+Although it may be time consuming and difficult to manage, installing from source has a few advantages:
+
+- Always stay updated, whether it be a security patch or a new feature.
+- Trim down the features while installing so as to suit your needs.
```

Comment:
What do you mean by this?

Reply:
I was thinking from a code perspective, they can go in and edit stuff to their needs; but in hindsight, I don't think there's really a need to include that at all. I'll take that bullet out.